packages/@aws-cdk-testing/framework-integ/test/aws-dynamodb/test/integ.dynamodb.policy.ts

```diff
@@ -12,25 +12,31 @@ export class TestStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
-    const doc = new iam.PolicyDocument({
-      statements: [
-        new iam.PolicyStatement({
-          actions: ['dynamodb:*'],
-          principals: [new iam.AccountRootPrincipal()],
-          resources: ['*'],
-        }),
-      ],
-    });
+    // const doc = new iam.PolicyDocument({
+    //   statements: [
+    //     new iam.PolicyStatement({
+    //       actions: ['dynamodb:*'],
+    //       principals: [new iam.AccountRootPrincipal()],
+    //       resources: ['*'],
+    //     }),
+    //   ],
+    // });
 
     this.table = new dynamodb.Table(this, 'TableTest1', {
       partitionKey: {
         name: 'id',
         type: dynamodb.AttributeType.STRING,
       },
       removalPolicy: RemovalPolicy.DESTROY,
-      resourcePolicy: doc,
+      // resourcePolicy: doc,
     });
 
+    this.table.addToResourcePolicy(new iam.PolicyStatement({
+      actions: ['dynamodb:GetItem'],
+      principals: [new iam.AccountRootPrincipal()],
+      resources: ['*'],
+    }));
+
```

Comment:
I think the change in this PR is already covered in the unit test. Integ test does not seem necessary to me because we are not using new L1 props. Is there anything the integ test would catch while the unit test cannot?